terraform/modules/apps/content-store/main.tf

```
@@ -8,22 +8,22 @@ terraform {
 }
 
 data "aws_secretsmanager_secret" "oauth_id" {
-  name = "content_store_app-OAUTH_ID"
+  name = "${var.draft ? "draft_content_store_app-OAUTH_ID" : "content_store_app-OAUTH_ID"}"
```

I'd recommend just substituting the name for now, like this. (I know that probably means re-inputting the secrets, sorry - shouldn't take more than a few mins though with a bit of command line and copy-paste.)

Or if you really don't fancy doing that then maybe just stick a TODO on this to say that the conditionals are a temporary hack until we get rid of these secrets. Just as long as the pattern doesn't start spreading around the codebase! 😅 

fyi - all of these secrets except for SENTRY_DSN will be going away in the near future (because using the signon-based auth for app-app traffic prevents us from bootstrapping a new environment without a ton of manual setup), and SENTRY_DSN is actually the same everywhere.

```suggestion
  name = "${var.service_name}-OAUTH_ID"
```

---

Hi, I don't mind using the service name (content-store) directly but I would have to create a local value (content_store) to have the right style?

---

I haven't done the secrets yet, was waiting for feedback.

---

The style of those names isn't really something that anyone thought about btw - it's left over from the discovery / proof-of-concept stuff from a while back.